scripts/set-default-contact/index.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+import request from 'request-promise-native';
+import express from 'express';
```

Comment:
Hmmm. I had trouble getting to this to work. I'd get this:

```
file:///home/mrjones/Documents/MedicMobile/cht-core/scripts/set-default-contact/index.js:1
const request = require('request-promise-native');                                              
                ^
                                                                                                                                                                                                 
ReferenceError: require is not defined in ES module scope, you can use import instead
```

the fix was to ...move to a different file suffix for CommonJS?  Seems exotic to the CHT:

> This file is being treated as an ES module because it has a '.js' file extension and '/home/mrjones/Documents/MedicMobile/cht-core/scripts/set-default-contact/package.json' contains "type": "mo
dule". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.

I'll submit it back to you any way and we can see where we end up!

Reply:
I think you have to remove the line with `"type": "module",` in your package.json

Reply:
Aha! That fixed the require errors - and I'm back to `.js`.  Committed in 2b230f9de !